server/sql/dev/create-tables-dev.sql

```diff
@@ -1,18 +1,47 @@
 \c socobo_dev;
 
+--
+--  SOCOBO USER
+--
+
+CREATE TABLE Socobo_User_Role (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR NOT NULL UNIQUE,
+  created NUMERIC NOT NULL,
+  lastModified NUMERIC NOT NULL CHECK (lastModified >= created)
+);
+
+CREATE TABLE Socobo_User_Provider (
+  id SERIAL PRIMARY KEY,
```

Comment:
I think it would be better to use the name (is already unique) as id. Else we always need to look into the db to figure out the current id. If we use the names (should be constant) as id we could always rely on the chosen name. We could also use the name directly in the frontend. If we use the id we would alway need to do a request for the providers to get the ids.